dashboard/test/models/contact_rollups_processed_test.rb

```diff
@@ -112,22 +118,73 @@ class ContactRollupsProcessedTest < ActiveSupport::TestCase
     contact_data = {
       'dashboard.pd_enrollments' => {
         'course' => [
-          {'value' => Pd::Workshop::COURSE_CSF},
-          {'value' => Pd::Workshop::COURSE_CSF},
-          {'value' => Pd::Workshop::COURSE_CSD},
+          {'value' => COURSE_CSF},
+          {'value' => COURSE_CSF},
+          {'value' => COURSE_CSD},
           {'value' => nil}
         ]
       }
     }
     # output should not contains nil or duplicate values, and should be sorted
     expected_output = {
-      professional_learning_enrolled: "#{Pd::Workshop::COURSE_CSD},#{Pd::Workshop::COURSE_CSF}"
+      professional_learning_enrolled: "#{COURSE_CSD},#{COURSE_CSF}"
     }
 
     output = ContactRollupsProcessed.extract_professional_learning_enrolled(contact_data)
     assert_equal expected_output, output
   end
 
+  test 'extract_professional_learning_attended' do
```

Comment:
Is your thought that we have tests for each processing step? Or that we add tests specific to the test when that test represents a new "type" of processing step?

Reply:
I add test for this attribute because multi-value type is quite tricky, which leads me to discover the problem fixed in https://github.com/code-dot-org/code-dot-org/pull/35472. 
For other multi-value or single-value fields, we may not need to add thorough tests. Just need to make sure the new code is exercised in at least 1 test and we can verify its result.